R/prepare_istr_company.R

```diff
@@ -7,29 +7,16 @@
 #' @param istr_comp A dataframe like the corresponding output at company level
 #'   from tiltIndicator.
 #' @param eco_inputs A dataframe like [ecoinvent_inputs]
+#' @param isic_tilt_map A dataframe like [isic_tilt_mapper]
 #'
 #' @return A dataframe that prepares the final output of ictr_company
 #'
 #' @export
-#'
-#' @examples
-#' library(tiltIndicator)
-#'
-#' company <- unnest_company(toy_sector_profile_upstream_output())
-#' product <- unnest_product(toy_sector_profile_upstream_output())
-#'
-#' prepare_istr_company(
-#'   company |> head(3),
-#'   product |> head(3),
-#'   ep_companies |> head(3),
-#'   ecoinvent_activities |> head(3),
-#'   matches_mapper |> head(3),
-#'   ecoinvent_inputs |> head(3)
```

Comment:
@maurolepore I had to remove these examples from the code because the `toy_sector_profile_upstream_output()` dataset does not contain the `isic_4digit` column required for joining the `isic_4digit_name` column through the `isic_tilt_mapper` file. 

At the moment the removal of these examples makes sense because we do not have right toy datasets for tiltIndicatorAfter. Hence, I would like you to decide how to proceed further with this issue. Should I create a separate issue for creating the toy datasets after merging this code? or you would be more interested in creating the new toy datasets first and write correct examples and then push this code?